src/containers/Accounts/AMM/AMMAccounts/AMMAccountHeader/AMMAccountHeader.tsx

```diff
@@ -6,6 +6,8 @@ import '../../../AccountHeader/balance-selector.scss'
 import { localizeBalance, localizeNumber } from '../../../../shared/utils'
 import Currency from '../../../../shared/components/Currency'
 
+const TRADING_FEE_TOTAL = 1000
```

Comment:
this should be pulled from utils.js instead

Reply:
It should use `formatTradingFee`

Reply:
done